src/main/scala/edu/ie3/simona/sim/setup/SetupHelper.scala

```diff
@@ -225,13 +226,16 @@ trait SetupHelper extends LazyLogging {
     * The provided type safe config must be able to be parsed as
     * [[SimonaConfig]], otherwise an exception is thrown
     *
+    * @param typeSafeConfig
+    *   All configuration parameters
     * @param simonaConfig
-    *   the configuration file
+    *   The configuration for SIMONA
     * @return
-    *   the resulting result file hierarchy
+    *   The resulting result file hierarchy
```

Comment:
Our new formatting rules also include full stops at the end of text lines, so it'd look something like this: 

```suggestion
    * [[SimonaConfig]], otherwise an exception is thrown.
    *
    * @param typeSafeConfig
    *   All configuration parameters.
    * @param simonaConfig
    *   The configuration for SIMONA.
    * @return
    *   The resulting result file hierarchy.
```

Just something I wanted to mention, because then the format does not have to be touched again :)